tracer/src/Datadog.Trace/Ci/TestModule.cs

```diff
@@ -283,10 +284,48 @@ public void Close()
     /// </summary>
     /// <param name="duration">Duration of the test module</param>
     public void Close(TimeSpan? duration)
```

Comment:
Should we add a note to the doc comments noting that they should use the async versions where possible?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3555/commits/108812b49008b0eb19bb72a51fae838e072a1024